pkgs/servers/http/nginx/modules.nix

```diff
@@ -209,10 +209,10 @@ in
       name = "modsecurity-nginx";
       owner = "SpiderLabs";
       repo = "ModSecurity-nginx";
-      rev = "v1.0.1";
-      sha256 = "0cbb3g3g4v6q5zc6an212ia5kjjad62bidnkm8b70i4qv1615pzf";
+      rev = "v1.0.2";
+      sha256 = "sha256-UXiitc3jZlgXlCsDPS+xEFLNRVgRbn8BCCXUEqAWlII=";
     };
-    inputs = [ pkgs.curl pkgs.geoip pkgs.libmodsecurity pkgs.libxml2 pkgs.lmdb pkgs.yajl ];
+    inputs = [ pkgs.curl pkgs.geoip pkgs.libmodsecurity.dev pkgs.libxml2 pkgs.lmdb pkgs.yajl ];
```

Comment:
Why does this one needs dev but the others don't?

Reply:
Don't need to include `dev`?. Can I remove it? There will be no problems with finding the header files  later?

Reply:
If there were, we would have them with the other packages as well, but normally dev outputs are used by default.

Reply:
I think you can drop it if this is list is just passed to `buildInputs`.

Reply:
Fixed.